ansible/playbooks/roles/video-stream-dump/handlers/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: restart streamdump target
+  become: true
+  systemd:
+    daemon_reload: true
+    name: streamdump.target
+    state: restarted
+
+- name: restart streamdump service
+  become: true
+  systemd:
+    daemon_reload: true
+    name: "streamdump-{{ item.host }}.service"
+    state: restarted
+  with_items: "video_stream_dump_services.results"
+  when: item | changed
```

Comment:
nit: this syntax is deprecated since Ansible 2.5 and will be removed in 2.9: https://docs.ansible.com/ansible/latest/porting_guides/porting_guide_2.5.html#jinja-tests-used-as-filters

Reply:
Ahh, thanks, will fix.